polkadot/node/core/approval-voting/src/tests.rs

```diff
@@ -1155,59 +1162,6 @@ fn subsystem_rejects_bad_assignment_ok_criteria() {
 	});
 }
 
-#[test]
-fn subsystem_rejects_bad_assignment_err_criteria() {
```

Comment:
Checkes are now down in approval-distribution, so I moved the test there as well.